cmd/porter/run.go

```diff
@@ -23,7 +23,7 @@ func buildRunCommand(p *porter.Porter) *cobra.Command {
 			if opts.rawAction == "" {
 				opts.rawAction = os.Getenv(config.EnvACTION)
 				if p.Debug {
-					fmt.Fprintf(p.Out, "DEBUG: defaulting action to %s (%s)\n", config.EnvACTION, opts.rawAction)
+					fmt.Fprintf(p.Err, "DEBUG: defaulting action to %s (%s)\n", config.EnvACTION, opts.rawAction)
```

Comment:
Curious as to the decision to print to stderr (if that is what `p.Err` equates to) for this informational/non-error debug message?

Reply:
Agree, but this can get fixed in a followup AFAIAMC

Reply:
Yeah, this wasn't something I originally intended to do but then the schema command kind of forced me to go down this route. Let me share how it came about and maybe you have ideas on if it was a good idea or if there is another way to deal with it?

When you run `porter schema --debug` because a mixin is acting up or something isn't right, if everything goes to stdout, the debug output is intermixed with the manifest json schema. Then it's impossible to get a clean schema out. By putting the debug statements into stderr, you can do this

`porter schema --debug 2> err.log`

so that you can see the manifest and the debug lines and manage them separately

Thoughts? OK to keep it like this for now? Better way to deal?

Reply:
Ah, I see.  This reasoning makes sense to me!  Maybe we can add that example to docs around schema/debugging.

Reply:
How about in the contributing guide? I can add a note about how to do logging there.

Reply:
Good idea 👍